nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,61 @@
+
+# Modes overview
+
+## Default (single schema, single catalog, single SQL Server instance)
+
+- Supported in Versions 1.0 and higher.
+- Has simple configuration and setup.
+- Doesn't require Distributed Transaction Coordinator (MS DTC).
+- The snapshot (backup) of the entire system state can be done by backing up a single database. It is especially useful if business data is also stored in the same database.
+- Can be monitored with ServiceControl.
+
+## Multi-schema
+
+- Supported in Versions 2.1 and higher.
+- Has simple configuration and setup.
+- Doesn't require Distributed Transaction Coordinator (MS DTC).
+- The snapshot (backup) of the entire system state is done by backing up a single database. It is especially useful if business data is also stored in the same database.
+- Enables security configuration on a schema level.
+- Can't be monitored with ServiceControl.
+
+## Multi-instance (and multi-catalog)
+
+- Supported in Versions 2.1 to 2.x, legacy support is offered in Versions 3.x, won't be supported directly in Versions 4.0 and higher.
+- Requires Distributed Transaction Coordinator, or using Outbox and storing business data in the same database as Outbox data.
+- Can't be monitored with ServiceControl.
+
+## Multi-instance with store-and-forward
+
+SQL Server transport does not support store-and-forward mechanism natively. It means that in *multi-instance* mode if a remote endpoint's infrastructure (e.g. DTC or SQL Server instance) is down, then messages can't be delivered. That makes this particular endpoint, and all endpoints depending on it, unavailable. 
+
+The problem can be addressed using [Outbox](/nservicebus/outbox/) feature. In addition Outbox can be used to avoid escalating transactions to DTC, given that each endpoint has its own database in which it stores input queue, error and audit queues, and the user data. That means it's not possible to avoid distributed transactions in case when any of the queues (including error and audit) is on a different SQL Server instance or catalog. 
+
+When using Outbox:
+
+ * Messages are not dispatched immediately after the `Send()` method is called. Instead, they are first stored in the Outbox table in the same database that the endpoint's persistence is using. After the handler logic completes successfully, the messages stored in the Outbox table are forwarded to their final destinations.
+ * If any of the forward operations fails, the message sending will be retried using the standard [retry mechanism](/nservicebus/errors/automatic-retries.md). This might result in sending some messages multiple times, which is known as `at-least-once` delivery. The Outbox feature performs automatically de-duplication of incoming messages based on their IDs, effectively providing `exactly-once` message delivery. The receiving endpoint should also be configured to use the outbox.
```

Comment:
>If any of the forward operations fails, the message sending will be retried using the standard [retry mechanism](/nservicebus/errors/automatic-retries.md). This might result in sending some messages multiple times, which is known as `at-least-once` delivery. The Outbox feature performs automatically de-duplication of incoming messages based on their IDs, effectively providing `exactly-once` message delivery. The receiving endpoint should also be configured to use the outbox.

How about:

If any of the forward operations fails, the message sending will be retried using the standard [retry mechanism](/nservicebus/errors/automatic-retries.md). Attempting to retry the forward operation may result in dispatching some messages multiple times. However, the Outbox feature automatically de-duplicates the incoming messages based on their IDs, therefore providing `exactly-once` message delivery. For this to work, the receiving endpoint should also be configured to use the outbox.